electron-app/src/components/AccountManagement.vue

```diff
@@ -164,6 +191,32 @@ export default class AccountManagement extends Vue {
 </script>
 
 <style lang="scss">
+@keyframes scrollLarge {
+  0% {
+    transform: rotate(-13deg) translateY(0);
+  }
+  100% {
+    transform: rotate(-13deg) translateY(-1200px);
+  }
+}
+
+.account_management__loading {
```

Comment:
nitpicky: but the rul for loading is already at 229 &__loading nested in .account_management in scss translates to .account_management__loading